tests/services/pids/providers/test_datacite_pid_provider.py

```diff
@@ -37,6 +39,19 @@ def record(location):
     return record
 
 
+@pytest.fixture(scope="function")
+def record_w_links(running_app, minimal_record):
+    """Creates an empty record."""
+    service = current_rdm_records.records_service
+    draft = service.create(system_identity, minimal_record)
+    record = service.publish(draft.id, system_identity)
+    record = current_rdm_records.records_service.read(
+        record["id"], system_identity
+    )
```

Comment:
why is the `read` needed? `publish` should already be returning the record?

Reply:
You are right! Thanks!